packages/app/src/manifest.json

```diff
@@ -8,26 +8,18 @@
     "256": "assets/connect-logo/Stacks256w.png",
     "512": "assets/connect-logo/Stacks512w.png"
   },
-  "content_security_policy": "script-src 'self'<% DEV_CSR %>; object-src 'none'; frame-src 'none'; frame-ancestors 'none';",
-  "permissions": [
-    "activeTab",
-    "storage"
-  ],
+  "content_security_policy": "script-src 'self'<% DEV_CSR %>; object-src <% DEV_OBJECT_SRC %>; frame-src 'none'; frame-ancestors 'none';",
+  "permissions": ["activeTab", "storage"],
   "manifest_version": 2,
   "background": {
-    "scripts": [
-      "background.js"
-    ],
+    "scripts": ["background.js"],
     "persistent": true
   },
-  "web_accessible_resources": [
-    "inpage.js",
-    "index.html"
-  ],
+  "web_accessible_resources": ["inpage.js"],
```

Comment:
@hstove this change seems to not work for me when I try to use the extension and auth with a new app. Have you tried this with a prod build?